pkg/quota/apis/quota/v1/conversion.go

```diff
@@ -40,15 +39,3 @@ func Convert_quota_ResourceQuotasStatusByNamespace_To_v1_ResourceQuotasStatusByN
 
 	return nil
 }
-
-func addConversionFuncs(scheme *runtime.Scheme) {
```

Comment:
this being unused seems like a bug, doesn't it?

Reply:
thought it will be captured by serialization test?